### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by Element118**
I went and kept track of typos. Here's my first batch of fixes mainly targeting the 3xx pages.

For page 370 (p-adic), the Mahler coefficients seem wrongly defined, I suspect it's f(k) instead of f(n).

### CODE REVIEW

File: tex/measure/measure-space.tex [hunk 1]
```diff
@@ -238,7 +238,7 @@ \section{Measure spaces}
 		\ii The axioms on $\mu$ can be weakened to
 		(i) $\mu(\varnothing) = 0$,
 		(ii) $\mu(A \sqcup B) = \mu(A) + \mu(B)$, and
-		(iii) for $A_1 \supset A_2 \supset \dots$
+		(iii) for $A_1 \subset A_2 \subset \dots$
```
Comment (vEnhance) [inline tex/measure/measure-space.tex]: I think \supset was correct? Since it's \bigcap after.

**Comment by vEnhance** [comment]
> For page 370 (p-adic), the Mahler coefficients seem wrongly defined, I suspect it's f(k) instead of f(n).

Yep. Will fix that too.

File: tex/measure/measure-space.tex (hunk 1 above)
Comment (vEnhance) [reply tex/measure/measure-space.tex]: Although I suppose I should be using `\supseteq` anyways to be consistent with the rest of the book.

Comment (Element118) [reply tex/measure/measure-space.tex]: Hmm, yeah, whoops, let's use \supseteq.

**Comment by Element118** [comment]
This batch is looking good, let's merge these changes when you're ready.

**Pull request merged**